src/Validator.php

```diff
@@ -52,13 +62,15 @@ public function __construct(Model $model)
     /**
      * Set one rule.
      *
+     * @param array<string|array<string|int|\Closure(string, mixed, list<mixed>, list<mixed>): bool>> $rules
```

Comment:
called from https://github.com/vlucas/valitron/blob/v1.4.11/src/Valitron/Validator.php#L1251

type LGTM, the last list can be maybe `list<string>`, untested, not sure what is passed

Reply:
yeah i was not sure too. that's why left mixed there to be sure.